Src/Plugins/Tests.Security/Validators/SEC101_102.AdoPatValidatorTests.cs

```diff
@@ -53,16 +53,17 @@ public void AzureDevOpsPersonalAccessTokenValidator_IsValidBasic()
                 bool performDynamicValidation = testCase.PerformDynamicValidation;
                 string failureLevel = testCase.FailureLevel;
                 string fingerprintText = null;
-                var groups = new Dictionary<string, FlexMatch>();
+                var groups = new Dictionary<string, FlexMatch>
```

Comment:
>groups [](http://example.com/codeflow?start=20&length=6)

why didn't we notice this problem before? looks like we're perhaps missing a key unit test somewhere?

Reply:
This was a strange test.
I removed a few things and started validating the expected count and state.